pkg/cluster/cluster.go

```diff
@@ -512,6 +554,21 @@ func (c *Cluster) Update(oldSpec, newSpec *acidv1.Postgresql) error {
 		}
 	}()
 
+	// check if pod resources were edited below the enforced minimum limits
+	if err := c.validateResources(&newSpec.Spec); err != nil {
+		err = fmt.Errorf("insufficient resource limits specified: %v", err)
+
+		isCPULimitSmaller, err2 := util.IsSmallerQuantity(newSpec.Spec.Resources.ResourceLimits.CPU, oldSpec.Spec.Resources.ResourceLimits.CPU)
```

Comment:
why check resource limits again ? isn't it caught by `err := c.validateResources(&newSpec.Spec);` at the line above ?

Reply:
moved the comment here and explain my intention a bit better